### PR TITLE
Design mode image and canvas placeholders

### PR DESCRIPTION
**Pull request by joshlory**
Mocks from the designer:

Implementation:

Continuation of https://github.com/code-dot-org/code-dot-org/pull/2844.

### CODE REVIEW

**Comment by davidsbailey** [comment]
Why does it say "Drag the Canvas"? I'm having trouble guessing what that could mean.

**Comment by joshlory** [comment]
Yeah, I didn't add the text to the placeholders yet.  It's tough to get right in CSS and I don't think it adds a ton of value.  Seems like something we could add later maybe.

File: apps/style/applab/skins/modern.scss [hunk 1]
```diff
@@ -30,16 +26,39 @@
...
+    border: 5px solid #fff;
+    outline: 1px solid #bdc3c7;
+    outline-offset: -1px;
+  }
+
+  img[src=''] {
+    background: #bdc3c7 url("/blockly/media/applab/placeholder_img.png") center center no-repeat;
+  }
+
+  canvas {
```
Comment (davidsbailey) [inline apps/style/applab/skins/modern.scss]: Is the canvas going to keep this background image/color once I start drawing on it? a screenshot of a non-empty canvas / image would help (or its fine as long as you're checking those cases)

**Comment by davidsbailey** [comment]
LGTM

File: apps/style/applab/skins/modern.scss (hunk 1 above)
Comment (joshlory) [inline apps/style/applab/skins/modern.scss]: Nope, this only shows up in Design mode (it gets hidden when the program is run).

**Pull request merged**